docs/SPEC.md

```diff
@@ -2240,31 +2240,31 @@ The group key of the resulting table is empty.
 
 #### Keys
 
-Keys lists the column labels of input tables.
-For each input table, it outputs a table with the same group key columns, plus a `_value` column containing the labels of the input table's columns.
-Each row in an output table contains the group key value and the label of one column of the input table.
-So, each output table has the same number of rows as the number of columns of the input table.
+Keys outputs the group key of input tables.
+For each input table, it outputs a table with the same group key columns, plus a `_value` column containing the labels of the input table's group key.
+Each row in an output table contains the group key value and the label of one column in the group key of the input table.
+So, each output table has the same number of rows as the size of the group key of the input table.
 
-Keys has the following parameters: 
+Keys has the following properties:
 
-| Name   | Type     | Description                                                                               |
-| ----   | ----     | -----------                                                                               |
-| except | []string | Expect is a list of keys not to include in the output. Defaults to `["_time", "_value"]`. |
+| Name   | Type   | Description                                                                  |
+| ----   | ----   | -----------                                                                  |
+| column | string | Name of the column used to store the group key labels. Defaults to `_value`. |
```

Comment:
nit, I have been following Go comment convention with these parameter comments. https://blog.golang.org/godoc-documenting-go-code Specifically the part where the comment starts with the name of the thing being commented and is a complete sentence. 

For example this could change to:

    Column is the name of the output column to store the group key labels. Defaults to `_value`.

Reply:
I missed this on the `columns` function review, its been merged already but if you could fix it in the PR that would be great. Its a small but important change.

Reply:
Uops, missed this comment!